strax/context.py

```diff
@@ -731,7 +740,7 @@ def __get_requested_plugins_from_cache(
                 continue
 
             requested_p = plugin.__copy__()
-            requested_p.run_id = run_id
+            requested_p.run_id = self.hyperrun_id(run_id)
```

Comment:
Isn't it just `_run_id`? Probably better to be consistent but maybe I'm too nitpicking

Reply:
What do you mean by `_run_id`?
`self.hyperrun_id` is used several times so I write a function.

Reply:
I meant that `_run_id = self.hyperrun_id(run_id)` was already calculated previously, so it doesn't harm to just use it?

Reply:
GitHub website can wrap the unchanged block of codes.

Reply:
The already calculated `_run_id` is from `def _set_plugin_config`. But the `requested_p.run_id = self.hyperrun_id(run_id)` is in `def 
 __get_requested_plugins_from_cache`.